docs/reference/method/MongoDBCollection-renameCollection.txt

```diff
@@ -0,0 +1,81 @@
+===========================
+MongoDB\\Collection::renameCollection()
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Collection::renameCollection()
+
+   Rename the collection.
+
+   .. code-block:: php
+
+      function renameCollection(string $newCollectionName, array $options = []): array|object
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBCollection-method-renameCollection-param.rst
+
+   The ``$options`` parameter supports the following options:
+
+   .. include:: /includes/apiargs/MongoDBCollection-method-renameCollection-option.rst
+
+Return Values
+-------------
+
+An array or object with the result document of the :manual:`renameCollection
+</reference/command/renameCollection>` command. The return type will depend on the
+``typeMap`` option.
+
+Errors/Exceptions
+-----------------
+
+.. include:: /includes/extracts/error-unsupportedexception.rst
+.. include:: /includes/extracts/error-invalidargumentexception.rst
+.. include:: /includes/extracts/error-driver-runtimeexception.rst
+
+Example
+-------
+
+The following operation renames the ``restaurants`` collection in the ``admin``
+database to ``zips``:
+
+.. code-block:: php
+
+   <?php
+
+   $collection = (new MongoDB\Client)->admin->restaurants;
+
+   $result = $collection->renameCollection('zips');
+
+   var_dump($result);
+
+The output would then resemble::
+
+   object(MongoDB\Model\BSONDocument)#9 (1) {
+     ["storage":"ArrayObject":private]=>
+     array(3) {
+       ["ns"]=>
+       string(16) "test.restaurants"
+       ["nIndexesWas"]=>
+       int(3)
+       ["ok"]=>
+       float(1)
+     }
+   }
```

Comment:
copied from `MongoDBCollection-dropCollection.txt`, I'm not sure what the correct output should be yet